_adr/80/index.adoc

```diff
@@ -12,11 +12,80 @@ In order to provide support to customers of a Managed Service, CEE require a mec
 state of the instance of the service for diagnostic purposes.  CEE will use this information in conjunction with the
 problem statement provided by the customer in order to efficiently investigate the problem and determine a solution.
 
-The data gathering mechanism must consider the issue of Personally Identifiable Information (PII) or other sensitive
-information and ensure that it is handled appropriately.
+The data gathering mechanism must consider the issue of
+https://source.redhat.com/departments/it/digitalsolutionsdelivery/enterprise-architecture/mojo_content/red_hat_data_classifications
```

Comment:
This is a reference to a private resource, let's avoid that.

Reply:
> This proposed ADR is interesting as it references extensively some Red Hat specific mechanism. I've started a conversation to see how we want to address this.

I see at least some of Backplane is open, https://github.com/openshift/managed-scripts, but how complete/usable the open source offering is for someone say running kube  I don't understand the Backplane landscape sufficiently well to answer that.